samples/WOCCatalog/WOCCatalog/AccelerateViewController.m

```diff
@@ -31,189 +31,172 @@
 #import "AccelerateViewController.h"
 #import "SingleImageViewController.h"
 
-@implementation AccelerateViewController
-
-int accelerateImageNumber = 1;
-int valueRed = 100, valueGreen = 100, valueBlue = 100;
-int convolveSize = 1;
-NSArray *indexPathArray;
-UIImage *img;
-UIImageView *imv;
-UISlider* redSlider;
-UISlider* greenSlider;
-UISlider* blueSlider;
-UISlider* convolveSlider;
-UILabel *rLabel, *gLabel, *bLabel, *boxLabel;
+static const double meanDivisor = 100;
 
+@implementation AccelerateViewController {
+    int _accelerateImageNumber = 1;
+    int _valueRed = 100;
+    int _valueGreen = 100;
+    int _valueBlue = 100;
+    int _convolveSize = 1;
```

Comment:
This would be build break.  You need to move initialization to the init or viewDidLoad method.